nav2_system_tests/src/updown/timeout.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```

Comment:
There is an existing gnu command `timeout` that seems to do exactly this. Does that command not work?

I don't see a clear copyright associated with this script.

Reply:
Deleted the timeout.sh script and using GNU timeout.